Makefile

```diff
@@ -110,6 +111,9 @@ generate-python-client: $(BUILD_FOLDER)
 generate-keys: $(BUILD_FOLDER)
 	cd tools && go run auth_keys_generator.go -keys-dir=$(BUILD_FOLDER)
 
+verify-generated-files: generate generate-from-swagger
+	${ROOT_DIR}/tools/verify-generated-files.sh
```

Comment:
Add `make format` before the validation it will fix import and lint issues

Reply:
Adding `make format` would be cheating the CI. We run this verification in order to verify if the committed changes are legitimate.

Reply:
The generated code is not formatted according to `goimports` so adding it the as part of generate target is fine.

Reply:
i wrote the comment in the wrong place, it should be part of make generate.

Reply:
Alright, I see. You're saying we push code without it being formatted.
SGTM

Reply:
I'm not sure what's going on ... I added this:
```patch
diff --git a/Makefile b/Makefile
index 6dc95683..a7ade2f5 100644
--- a/Makefile
+++ b/Makefile
@@ -85,6 +85,7 @@ format:
 
 generate:
        go generate $(shell go list ./... | grep -v 'assisted-service/models\|assisted-service/client\|assisted-service/restapi')
+       $(MAKE) format
 
 generate-from-swagger: generate-go-client generate-go-server
 
```

But now `skipper make generate` fails on `goimports`:
```
goimports -w -l cmd/ internal/ subsystem/ pkg/ assisted-iso-create/
internal/cluster/mock_cluster_api.go
internal/connectivity/mock_connectivity_validator.go
internal/events/mock_event.go
internal/hardware/mock_validator.go
internal/host/mock_host_api.go
internal/host/mock_instruction_api.go
internal/metrics/mock_netricsManager_api.go
internal/network/mock_ntp_utils.go
pkg/generator/mock_install_config.go
pkg/k8sclient/mock_k8sclient.go
pkg/leader/mock_leader_elector.go
pkg/ocm/mock_authorization.go
pkg/ocm/mock_pullsecret_auth.go
pkg/s3wrapper/mock_s3iface.go
pkg/s3wrapper/mock_s3wrapper.go
make[1]: *** [Makefile:79: format] Error 1
make[1]: Leaving directory '/home/ncarboni/Source/assisted-service'
make: *** [Makefile:88: generate] Error 2
```

But if I run `skipper make format`, it succeeds but also doesn't clean up any of the imports issues:
```
$ skipper make format
[skipper] No build container tag was provided
...
Successfully built e7f24fddf102
Successfully tagged assisted-service-build:latest
goimports -w -l cmd/ internal/ subsystem/ pkg/ assisted-iso-create/
gofmt -w -l cmd/ internal/ subsystem/ pkg/ assisted-iso-create/
$ git status -s
 M Makefile
 M internal/cluster/mock_cluster_api.go
 M internal/connectivity/mock_connectivity_validator.go
 M internal/events/mock_event.go
 M internal/hardware/mock_validator.go
 M internal/host/mock_host_api.go
 M internal/host/mock_instruction_api.go
 M internal/metrics/mock_netricsManager_api.go
 M internal/network/mock_ntp_utils.go
 M pkg/generator/mock_install_config.go
 M pkg/k8sclient/mock_k8sclient.go
 M pkg/leader/mock_leader_elector.go
 M pkg/ocm/mock_authorization.go
 M pkg/ocm/mock_pullsecret_auth.go
 M pkg/s3wrapper/mock_s3iface.go
 M pkg/s3wrapper/mock_s3wrapper.go
```
:confused:

Reply:
IMO we don't really need to upload the mock files, we can generate them each time we run the testing.

Reply:
What do you mean by "upload", commit?
That would be quite a large change for just this ...

Reply:
Strange, i just added this code:
```
 generate-from-swagger: generate-go-client generate-go-server
+       make format
```
and everything worked

Reply:
Yeah, that's a different generate command. The one that builds the mocks and has the import ordering issue is `generate`, not `generate-from-swagger`

Reply:
Ohh we were talking about generate from swagger no? i put the command after generate from swagger.

Reply:
This PR concerns both.
The issue with import ordering only exists for `generate` I think.